versioned_docs/version-3.10/operator-guide/cd/auto-stable-trigger-type.md

```diff
@@ -0,0 +1,42 @@
+# Improve Application Stability With Consequent Deployment
+
+When application is maintained by several developers at a time, it can get several updates a day, which might interfere with each other, resulting in application instability. To eliminate this need, **auto-stable** trigger type was introduced in KubeRocketCI.
```

Comment:
It doesn't depend on the number of developers on the project, it depends on the approach to deployment.